int8 num
string name_R
string name_L
float64 position_R
float64 position_L
float64 velocity_R
float64 velocity_L